cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1109,6 +1109,7 @@ def _add_efs_storage(self, id: str, shared_efs: SharedEfs):
             shared_efs.encryption_in_transit
         )
         self.shared_storage_attributes[SharedStorageType.EFS]["IamAuthorizations"].append(shared_efs.iam_authorization)
+        self.shared_storage_attributes[SharedStorageType.EFS]["AccesspointIds"].append(shared_efs.accesspoint_id)
```

Comment:
Here and in other files, we are assuming that more than one access point can be specified for a given file system (AccesspointsIds rather than AccesspointId).

It's true that a file system can support multiple access points (https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html#efs-access-poiont-create), but the mount action is always done using a single access point (https://docs.aws.amazon.com/efs/latest/ug/mounting-access-points.html).

So please, let's assume to specify only one access point on the cluster config.

Reply:
I think this is actually a per filesystem array, similar to the encryption in transit array

Reply:
Yes, it's actually an array of all the separate FS in config. I just aligned naming and structure to the other properties which are also named in plural.